packages/anomaly-detection-store/README.md

```diff
@@ -0,0 +1,55 @@
+<!--suppress HtmlDeprecatedAttribute -->
+<h1 align="center">
+  <br>
+  <a href="https://www.sphereon.com"><img src="https://sphereon.com/content/themes/sphereon/assets/img/logo.svg" alt="Sphereon" width="400"></a>
+  <br>Anomaly Detection Store (Typescript) 
+  <br>
+</h1>
+
+---
+
+**Warning: This package still is in very early development. Breaking changes without notice will happen at this point!**
+
+---
+
+A Veramo anomaly detection store plugin. This plugin stores the result of the location lookup which is part of the anomaly detection plugin
```

Comment:
Please stop using the word Veramo everywhere. This really is not a Veramo plugin

Reply:
Replaced Veramo with Sphereon